study-builder/studies/pancan/study-events.conf

```diff
@@ -55,11 +25,7 @@
         "type": "ACTIVITY_INSTANCE_CREATION",
         "activityCode": ${id.act.parental}
       },
-      "preconditionExpr": """
-        user.studies["cmi-pancan"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("CHILD_DIAGNOSED")
-        && user.studies["cmi-pancan"].forms["PREQUAL"].questions["CHILD_CURRENT_AGE"].isAnswered()
-        && user.studies["cmi-pancan"].forms["PREQUAL"].questions["CHILD_CURRENT_AGE"].answers.value() < 7)
-      """,
+      "preconditionExpr": ${_pex.is_child_only}"&&"${_pex.is_age_parental},
```

Comment:
we need additional check like 
&& user.studies["cmi-pancan"].forms["PREQUAL"].questions["CHILD_CURRENT_AGE"].isAnswered()
to avoid running into error in STUDY_REDIRECT workflow cases. 
In prequal select "child" and choose "brain"  (existing cmi). once submitted we run into pex evaluation issue

Reply:
Okay, understood. I think what we want here is "not redirected" but "age answered" is a good alternative.

Reply:
Fixed